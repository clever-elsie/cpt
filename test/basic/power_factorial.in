let pow1 = 2^3;
let pow2 = 2**4;
let fact1 = 5!;
let fact2 = (3+2)!;

\print(pow1, pow2, fact1, fact2); 