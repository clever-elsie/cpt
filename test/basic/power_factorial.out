8
16
120
120
4
